nix/sources.json

```diff
@@ -12,7 +12,7 @@
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     },
     "cardano-node": {
-        "branch": "master",
+        "branch": "cad-1347-bump-genesis-staking",
```

Comment:
Is it normal that we use (temporary?) branch here instead of `master`?

Reply:
The update to the new version of `cardano-node` / `ouroboros-network` is going to be non-trivial, so it probably makes sense to separate that work.